contracts/JoepegExchange.sol

```diff
@@ -778,6 +859,17 @@ contract JoepegExchange is
         return (protocolFee * _amount) / PERCENTAGE_PRECISION;
     }
 
+    function _validateMakerOrder(OrderTypes.MakerOrder calldata makerOrder)
+        internal
+        view
+        returns (bool)
+    {
+        return
+            !_isUserOrderNonceExecutedOrCancelled[makerOrder.signer][
+                makerOrder.nonce
+            ] && makerOrder.nonce >= userMinOrderNonce[makerOrder.signer];
+    }
```

Comment:
perhaps we should rename this method to `_checkMakerOrderNotExpired` to make its difference clear w/ `_validateOrder` and also suggest that its only validating one part of the maker order

Reply:
yeah sounds good, done!